CoursesManager.UI/CoursesManager.UI.csproj

```diff
@@ -26,10 +27,10 @@
 	</ItemGroup>
 
 	<!-- Conditional inclusion of the .env file -->
-	<ItemGroup Condition="Exists('.env')">
-		<Content Include=".env">
-			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
-		</Content>
+	<ItemGroup>
+	  <Content Include=".env">
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+	  </Content>
```

Comment:
Kun je dit terugdraaien? Dit is van belang voor de pipeline.

Reply:
Akkoord, gwn weghalen? of

Reply:
k zeg wel akkoord, maar wat moet er precies op die plek komen te staan? lol